activemodel/lib/active_model/validations/length.rb

```diff
@@ -29,7 +29,7 @@ def initialize(options)
       def check_validity!
         keys = CHECKS.keys & options.keys
 
-        if keys.empty?
+        if keys.empty? || options.keys.include?(:minium)
```

Comment:
How about these?

We add the possible `options.keys` into `RESERVED_OPTIONS` above.

and we can do something like this

```ruby
unless options.keys.all? { |k| RESERVED_OPTIONS.include?(k) }
  raise ArgumentError, "Range unspecified. Specify the :in, :within, :maximum, :minimum, or :is option."
end

keys = CHECKS.keys & options.keys
```